tests/runner.py

```diff
@@ -808,8 +808,8 @@ def assertNotExists(self, filename, msg=None):
 
   # Tests that the given two paths are identical, modulo path delimiters. E.g. "C:/foo" is equal to "C:\foo".
   def assertPathsIdentical(self, path1, path2):
-    path1 = path1.replace('\\', '/').replace('//', '/')
-    path2 = path2.replace('\\', '/').replace('//', '/')
```

Comment:
I'm not totally sure about this.. but if not tests fail then I guess its fine.

Reply:
As I said in the description, this is what was hiding the problem in the first place - `assertPathsIdentical` shouldn't fix double-slashes itself, as real consumers also wouldn't.